lib/studies/reply/inbox.dart

```diff
@@ -9,7 +9,7 @@ class InboxPage extends StatelessWidget {
   Widget build(BuildContext context) {
     final isDesktop = isDisplayDesktop(context);
 
-    return Center(
+    return SafeArea(
```

Comment:
By removing the center widget it might not be centered anymore. This will be noticeable on Desktop. You might need to wrap the row widget with a center widget.